contracts/DaiHrd.sol

```diff
@@ -486,38 +479,38 @@ contract DaiHrd is ERC777, MakerFunctions {
 		}
 	}
 
-	function deposit(uint256 attodai) external returns(uint256 depositedAttopot) {
+	function deposit(uint256 attodai) external returns(uint256 attodaiHrd) {
```

Comment:
We seemed to be inconsistent about return value naming, so I tried to clean these up to all just be units.  The events include `minted` and `burned` prefixes which I like, but for return values it feels unnecessary and misleading in some cases.

Also, I intentionally used a different variable name than the function to sort-of force an explicit return statement (I hate this aspect of Solidity).

Reply:
Great.